README.md

```diff
@@ -21,12 +21,28 @@ We welcome any issues and Pull Requests to improve the Emulator UI. The followin
 
 (If you're looking forward to use the Emulator UI as a user, please refer to the Usage section above.)
 
+### Setting up your development environment
+
+Clone the repo and install any dependencies:
+
+```bash
+git clone git@github.com:firebase/firebase-tools-ui.git
+cd firebase-tools
+npm install # must be run the first time you clone
+```
+
 ### Start the Development Server
 
+Select the emulators that you will be interacting with in the UI by executing the command below and following the inline prompts:
```

Comment:
nit: Now you can remove this paragraph and `init` since it's already done

Reply:
Ah whoops! Thanks for catching that!